_data/events/HEP-Training-Challenge-Nov2021.yml

```diff
@@ -0,0 +1,10 @@
+name: HSF IRIS-HEP Training Challenge 2
+startdate: 2021-11-01
+enddate: 2021-11-01
+location: Virtual
+meetingurl: https://indico.cern.ch/event/1088551/
+image:
+description: Our goal is to collect information from the different experiments about their training activities, experiences, and needs. This will mostly be done in a long google document before the workshop.
+During the workshop itself, we will then discuss and draw conclusions from the document. If you are passionate about software training, education or sustainability, please register for the workshop and help us to complete the information about your organization.
```

Comment:
```suggestion
description: >
  Our goal is to collect information from the different experiments about their training activities, experiences, and needs. This will mostly be done in a long google document before the workshop.
  During the workshop itself, we will then discuss and draw conclusions from the document. If you are passionate about software training, education or sustainability, please register for the workshop and help us to complete the information about your organization.
```